chef_master/source/dk_windows.rst

```diff
@@ -1,101 +1,195 @@
 
-On new Windows machines, PowerShell scripts will not work until an administrator runs the following command:
+Next, add the output of ``chef shell-init powershell`` to your ``$PROFILE`` (This command does not print the output onto the screen):
 
-.. code-block:: bash
+.. code-block:: shell
 
-   Set-ExecutionPolicy RemoteSigned
+   PS C:\Users\<username>> chef shell-init powershell | Add-Content $PROFILE
 
-.. end_tag
+Once initialized, your paths and environment variables are set up correctly for every new PowerShell session. See your environment settings by using ``cat $PROFILE``:
+
+.. code-block:: shell
+
+    PS C:\Users|<username>> cat $PROFILE
+    $env:PATH="C:/opscode/chefdk/bin;C:/Users/<username>/AppData/Local/chefdk/gem/ruby/2.5.0/bin;C:/opscode/chefdk/embedded/bin;C:/opscode/chefdk/embedded/git/usr/bin"
+    $env:GEM_ROOT="C:/opscode/chefdk/embedded/lib/ruby/gems/2.5.0"
+    $env:GEM_HOME="C:/Users/<username>/AppData/Local/chefdk/gem/ruby/2.5.0"
+    $env:GEM_PATH="C:/Users/<username>/AppData/Local/chefdk/gem/ruby/2.5.0;C:/opscode/chefdk/embedded/lib/ruby/gems/2.5.0"
 
 Spaces and Directories
------------------------------------------------------
++++++++++++++++++++++++++++++++++++++++++++++++++++++
 .. tag windows_spaces_and_directories
 
-Directories that are used by Chef on Windows cannot have spaces. For example, ``C:\Users\Steven Danno`` will not work, but ``C:\Users\StevenDanno`` will. Because of this, the ``knife supermarket install`` subcommand will fail if the directory contains a space.
+Directories that are used by Chef on Windows cannot have spaces. For example, ``C:\Users\User Name`` will not work, but ``C:\Users\UserName`` will. Because of this, the ``knife supermarket install`` subcommand will fail if the directory contains a space.
 
 .. end_tag
 
 Top-level Directory Names
------------------------------------------------------
++++++++++++++++++++++++++++++++++++++++++++++++++++++
 .. tag windows_top_level_directory_names
 
 Windows will throw errors when path name lengths are too long. For this reason, it's often helpful to use a very short top-level directory, much like what is done in UNIX and Linux. For example, Chef uses ``/opt/`` to install the Chef development kit on macOS. A similar approach can be done on Microsoft Windows, by creating a top-level directory with a short name. For example: ``C:\chef``.
 
 .. end_tag
+
+Install a Code Editor
+-------------------------------------------------------
+A good visual code editor is not a requirement for working with Chef, but a good code editor can save you time.
+A code editor should support the following: themes, plugins, snippets, syntax Ruby code coloring/highlighting, multiple cursors, a tree view of the entire folder/repository you are working with, and a Git integration.
+
+These are a few common editors: 
+
+   * Visual Studio Code (free/open source) - http://code.visualstudio.com
```

Comment:
Could also point to the Chef extension here? https://marketplace.visualstudio.com/items?itemName=Pendrica.Chef

Reply:
That's a good idea.